docs/cli/kops_edit.md

```diff
@@ -5,7 +5,7 @@ Edit clusters and other resources.
 ### Synopsis
 
 
-Edit a resource configuration. This command changes the cloud specification in the registry. 
+Edit a resource configuration. This command changes the cloud configuration in the registry. 
```

Comment:
This could be "desired configuration" - it's not really "cloud specification" anyway...